app/src/main/res/values/colors.xml

```diff
@@ -8,4 +8,12 @@
   <color name="secondaryDarkColor">#c30000</color>
   <color name="primaryTextColor">#DF000000</color>
   <color name="secondaryTextColor">#8A000000</color>
+
+  <!--Card backgrounds-->
+  <color name="cardBackgroundColor1">#9C27B0</color>
+  <color name="cardBackgroundColor2">#2196F3</color>
+  <color name="cardBackgroundColor3">#8BC34A</color>
+  <color name="cardBackgroundColor4">#FF9800</color>
+  <color name="cardBackgroundColor5">#FF7043</color>
```

Comment:
There is a better way to do it, you can use a resource array (an array of colors).